include/EditView/Panels.js

```diff
@@ -10,7 +10,7 @@
  * OF NON INFRINGEMENT OF THIRD PARTY RIGHTS.
  *
  * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * ANY WARRANTY; without even the mplied warranty of MERCHANTABILITY or FITNESS
```

Comment:
Hi Jerzy - https://suitecrm.com/wiki/index.php/Coding_Standards#License

No, this typo was added by the commit - there is no typo on the Coding Standards.  Update this.